### PR TITLE
LEP-273 refactor and move methods is_student? and in_work? from PersonWrapper

### PR DESCRIPTION
**Pull request by ripan**
[LEP-273](https://dsdmoj.atlassian.net/browse/LEP-273)

- Simplify PersonWrapper class and remove is_student? and in_work? methods

[LEP-273]: https://dsdmoj.atlassian.net/browse/LEP-273?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/services/calculators/multiple_employments_calculator.rb [hunk 1]
```diff
@@ -4,14 +4,14 @@ class << self
...
-          employment: DummyEmploymentFigures.new(0, 0, 0, 0),
+          employment: DummyEmploymentFigures.new(0, 0, 0, 0, 0),
           result: EmploymentIncomeCalculator::Result.new(fixed_employment_allowance: fixed_employment_allowance(submission_date)),
         )
       end
 
     private
 
-      DummyEmploymentFigures = Data.define(:monthly_gross_income, :monthly_benefits_in_kind, :monthly_tax, :monthly_national_insurance)
+      DummyEmploymentFigures = Data.define(:monthly_gross_income, :monthly_benefits_in_kind, :monthly_tax, :monthly_national_insurance, :actively_working?)
```
Comment (starswan) [inline app/services/calculators/multiple_employments_calculator.rb:14]: Think this should have an actively_working? method that always returns false? (not zero)

File: app/services/workflows/non_passported_workflow.rb [hunk 2]
```diff
@@ -145,16 +145,15 @@ def collate_and_assess_gross_income(assessment:, self_employments:, partner_self
       # TODO: make the Collators::DisposableIncomeCollator increment/sum to existing values so order of "collation" becomes unimportant
       def partner_disposable_income_assessment(assessment:, gross_income_subtotals:, applicant_person_data:, partner_person_data:)
         applicant = PersonWrapper.new is_single: false,
-                                      submission_date: assessment.submission_date,
-                                      person_data: applicant_person_data,
-                                      employments: assessment.employments,
-                                      gross_income_summary: assessment.applicant_gross_income_summary
+                                      person_data: applicant_person_data
```
Comment (starswan) [inline app/services/workflows/non_passported_workflow.rb:148]: think it's still to much to pass the whole person_data in here when all we want is the dependants?

File: spec/services/calculators/childcare_eligibility_calculator_spec.rb [hunk 3]
```diff
@@ -4,58 +4,76 @@ module Calculators
...
-        let(:partner) { OpenStruct.new(in_work?: true, is_student?: false) }
+        context "with adult dependants, an employed applicant and no partner" do
+          let(:dependants) { [OpenStruct.new(becomes_16_on: submission_date - 1.year)] }
 
-        it "returns true" do
-          expect(calculated_result).to eq true
+          it "returns true" do
+            expect(calculated_result).to eq false
+          end
         end
```
Comment (starswan) [inline spec/services/calculators/childcare_eligibility_calculator_spec.rb:51]: This is strange - you've appear to have added added 'returns true' when it is clearly false. Finding these changes hard to follow - are they strictly necessary?

Comment (ripan) [reply spec/services/calculators/childcare_eligibility_calculator_spec.rb]: Example description updated. I fixed the existing test and refactored a bit. Any suggestion what else we should add here?

**Review by starswan — APPROVED** [review]
